### PR TITLE
Add basic support for rendering code and other preformatted text

### LINKED ISSUE #236: "Code" type outputs, with syntax highlighting
so that output checkers can review code that needs to be released (or, minimal version, treat some code extensions as text so we display them without syntax highlighting)

requested by users

### PR DESCRIPTION
**Pull request by bloodearnest**
Works by having a list of file extensions to render.

Fixes #236

### CODE REVIEW

File: assets/src/scripts/_utils.js [hunk 1]
```diff
@@ -11,13 +11,14 @@ export const isCsv = (ext) => ext.toLowerCase() === "csv";
 export const isImg = (ext) =>
   ["gif", "jpg", "jpeg", "png", "svg"].includes(ext.toLowerCase());
 
+export const TEXT_TYPES = ["txt", "md", "py", "yaml", "R", "log"];
 /**
  * Confirm if file is .txt
  * @param {string} ext - file extension
  * @returns {boolean}
  */
 export function isTxt(ext) {
```
Comment (lucyb) [inline assets/src/scripts/_utils.js:20]: I'd be tempted to rename the function as `isText` to make it clearer that it's not just checking for a .txt extension.

File: assets/src/scripts/_utils.js [hunk 2]
```diff
@@ -11,13 +11,14 @@ export const isCsv = (ext) => ext.toLowerCase() === "csv";
 export const isImg = (ext) =>
   ["gif", "jpg", "jpeg", "png", "svg"].includes(ext.toLowerCase());
 
+export const TEXT_TYPES = ["txt", "md", "py", "yaml", "R", "log"];
 /**
  * Confirm if file is .txt
```
Comment (lucyb) [inline assets/src/scripts/_utils.js:16]: This code comment is now out of date, as it's no longer just checking for ".txt"

**Review by lucyb — APPROVED** [review]

File: assets/src/scripts/_utils.js (hunk 1 above)
Comment (bloodearnest) [reply assets/src/scripts/_utils.js:20]: Good idea, will do

File: assets/src/scripts/_utils.js (hunk 2 above)
Comment (bloodearnest) [reply assets/src/scripts/_utils.js:16]: Good catch

**Pull request merged**